clippy_lints/src/comparison_chain.rs

```diff
@@ -73,7 +73,7 @@ impl<'tcx> LateLintPass<'tcx> for ComparisonChain {
         }
 
         // Check that there exists at least one explicit else condition
-        let (conds, _) = if_sequence(expr);
+        let conds = if_sequence(expr).0;
         if conds.len() < 2 {
             return;
         }
```

Comment:
Hmm, my two cents;
I would not see this as an improvement in my own code. :/

If for example we do some code changes and it turns out we need the second return type of `if_sequence()` as well, we'd have to revert back to `(conds, ..)` which is more work than just replacing `_` with a variable binding.